### PR TITLE
feat: add `safeupdate` to `shared_preload_libraries`

### PR DESCRIPTION
**Pull request by soedirgo**
## What kind of change does this PR introduce?

Feature

## What is the current behavior?

pg_safeupdate can't be used with the `postgres` role since setting `[shared/session]_preload_libraries` requires superuser.

## What is the new behavior?

Allow using pg_safeupdate with the `postgres` role by setting `safeupdate.enabled = on`.

### CODE REVIEW

File: ansible/tasks/postgres-extensions/09-pg-safeupdate.yml [hunk 1]
```diff
@@ -28,3 +28,10 @@
   file:
     state: absent
     path: /tmp/pg-safeupdate-{{ pg_safeupdate_release }}
+
+- name: pg-safeupdate - set safeupdate.enabled
+  become: yes
+  lineinfile:
+    path: /etc/postgresql/postgresql.conf
+    state: present
+    line: safeupdate.enabled = off
```
Comment (darora) [inline ansible/tasks/postgres-extensions/09-pg-safeupdate.yml]: Why here instead of including in the base template?

Comment (soedirgo) [reply ansible/tasks/postgres-extensions/09-pg-safeupdate.yml]: We do this for other extensions (`cron.database_name`, `auto_explain.log_min_duration`, etc.) so just did the same here

Comment (soedirgo) [reply ansible/tasks/postgres-extensions/09-pg-safeupdate.yml]: I don't see a problem with including it in the base template though... going to smoke test that

Comment (soedirgo) [reply ansible/tasks/postgres-extensions/09-pg-safeupdate.yml]: Seems to work fine - updated the PR

**Review by steve-chavez — APPROVED** [review]

**Review by darora — APPROVED** [review]

**Pull request merged**